app/experimenter/nimbus-ui/src/components/FormOverview/index.tsx

```diff
@@ -120,233 +130,217 @@ const FormOverview = ({
-                risk mitigation checklist
-              </LinkExternal>{" "}
-              to make a copy and add the link above
-            </Form.Text>
-            <FormErrors name="riskMitigationLink" />
-          </Form.Group>
+        {experiment && (
+          <>
+            <Form.Group controlId="publicDescription">
+              <Form.Label className="d-flex align-items-center">
+                Public description
+                {isMissingField!("public_description") && (
+                  <InlineErrorIcon
+                    name="description"
+                    message="Public description cannot be blank"
+                  />
+                )}
+              </Form.Label>
+              <Form.Control
+                as="textarea"
+                rows={3}
+                {...formControlAttrs("publicDescription", {})}
+              />
+              <Form.Text className="text-muted">
+                This description will be public to users on about:studies
+              </Form.Text>
+              <FormErrors name="publicDescription" />
+            </Form.Group>
 
-          <Form.Group controlId="documentationLinks">
-            <Form.Label>
-              Additional Links
-              <Info
-                data-tip={DOCUMENTATION_LINKS_TOOLTIP}
-                data-testid="tooltip-documentation-links"
-                width="20"
-                height="20"
-                className="ml-1"
+            <Form.Group controlId="riskMitigationLink">
+              <Form.Label className="d-flex align-items-center">
+                Risk Mitigation Checklist Link
+                {isMissingField!("risk_mitigation_link") && (
+                  <InlineErrorIcon
+                    name="risk_mitigation_link"
+                    message="A Risk Mitigation Checklist is required"
+                  />
+                )}
+              </Form.Label>
+              <Form.Control
+                {...formControlAttrs("riskMitigationLink", {})}
+                type="url"
               />
-              <ReactTooltip />
-            </Form.Label>
-            <div>
-              {defaultValues.documentationLinks.map((docLink, docIdx) => (
-                <Form.Group
-                  className="mb-0"
-                  data-testid="DocumentationLink"
-                  key={`doc-link-${docIdx}`}
+              <Form.Text className="text-muted">
+                Go to the{" "}
+                <LinkExternal href={EXTERNAL_URLS.RISK_MITIGATION_TEMPLATE_DOC}>
+                  risk mitigation checklist
+                </LinkExternal>{" "}
+                to make a copy and add the link above
+              </Form.Text>
+              <FormErrors name="riskMitigationLink" />
+            </Form.Group>
+
+            <Form.Group controlId="documentationLinks">
+              <Form.Label>
+                Additional Links
+                <Info
+                  data-tip={DOCUMENTATION_LINKS_TOOLTIP}
```

Comment:
This is definitely not blocking, but I feel like we don't need this tooltip anymore since changing the link type to a drop-down. Would of course be an @AnaMedinac question though.

Reply:
Slacked her

Reply:
Hey! Agreed - I don't think we need the tooltip anymore.